Rakefile

```diff
@@ -4,6 +4,8 @@ require 'motion/project/template/ios'
 require 'bundler'
 Bundler.require(:development)
 require 'ProMotion'
+# Temporary - so we can use edge version of the gem
+require 'motion_print'
```

Comment:
just so it includes the proper version, also removed when we have the dependency linked to a released version.